jujugui/static/gui/src/app/init.js

```diff
@@ -333,11 +333,45 @@ class GUIApp {
     this.topology = this._renderTopology();
 
     /**
-      Once login is complete this will hold the connection to the controller
+      Once login is complete this will hold the connection to the controller.
       @type {Object}
     */
     this.controllerConnection = null;
 
+    /**
+      Once login is complete this will hold the connection to the model.
+      @type {Object}
+    */
+    this.modelConnection = null;
+
+    /**
+      The model uuid that is currently connected to.
+      @type {string}
+    */
+    this.activeModelUUID = null;
```

Comment:
Is there a reason you added this instead of  using `this.modelUUID`?

Reply:
I wanted to leave the old code in until it was all done so there was a name collision. I also wanted the name to be a little more explicit.

Reply:
OK, we'll just have to remember to consolidate these afterwards if we can, would be good to not have too many places to get out of sync.

Reply:
Yeah that's a good point, maybe I'll remove some of that stuff in this branch too so we don't have too much cruft left around.

Reply:
I have modified all of the uses of  `modelUUID` to instead use the `activeModelUUID` property.

Reply:
Thanks!